keyboards/moonlander/matrix.c

```diff
@@ -87,7 +87,11 @@ void matrix_init_custom(void) {
     setPinInputLow(A7);
     setPinInputLow(B0);
 
+    #ifdef HALFMOON
+    is_launching = true;
+    #else
     mcp23018_init();
+    #endif
```

Comment:
Actually, `mcp23018_init()` doesn't set is_launcing to false, only to true.  So we can just set it to true here and optionally call `mcp32018_init()`

```suggestion
    is_launching = true;
#    ifdef HALFMOON
    mcp23018_init();
#    endif
```

(in fact, we could just set `is_launching` to true by default)